pyta_server/static/test/testconfig.pylintrc

```diff
@@ -0,0 +1,581 @@
+[MASTER]
```

Comment:
Rather than this file, use the default `.pylintrc` that comes with python_ta (it should be under `python_ta/python_ta` in the repo.